main.py

```diff
@@ -1028,41 +1010,15 @@ async def dark_market_cutscene(interaction):
-                                                            .replace("{time}", caught_time[:-1]) + suffix_string,
-                                                view=view,
-                                                thread=discord.Object(message.channel.id),
-                                                allowed_mentions=discord.AllowedMentions.none())
-                        else:
-                            await send_target.send(coughstring.replace("{username}", message.author.name.replace("_", "\\_"))
-                                                            .replace("{emoji}", str(icon))
-                                                            .replace("{type}", le_emoji)
-                                                            .replace("{count}", str(add_cat(message.guild.id, message.author.id, le_emoji, silly_amount)))
-                                                            .replace("{time}", caught_time[:-1]) + suffix_string,
-                                                thread=discord.Object(message.channel.id),
-                                                allowed_mentions=discord.AllowedMentions.none())
-                    else:
-                        if view:
-                            await send_target.send(coughstring.replace("{username}", message.author.name.replace("_", "\\_"))
-                                                            .replace("{emoji}", str(icon))
-                                                            .replace("{type}", le_emoji)
-                                                            .replace("{count}", str(add_cat(message.guild.id, message.author.id, le_emoji, silly_amount)))
-                                                            .replace("{time}", caught_time[:-1]) + suffix_string,
-                                                view=view,
-                                                allowed_mentions=discord.AllowedMentions.none())
-                        else:
-                            await send_target.send(coughstring.replace("{username}", message.author.name.replace("_", "\\_"))
+                    await send_target.send(coughstring.replace("{username}", message.author.name.replace("_", "\\_"))
                                                             .replace("{emoji}", str(icon))
                                                             .replace("{type}", le_emoji)
                                                             .replace("{count}", str(add_cat(message.guild.id, message.author.id, le_emoji, silly_amount)))
                                                             .replace("{time}", caught_time[:-1]) + suffix_string,
+                                                *[view=view] if view else [],
+                                                *[thread=discord.Object(message.channel.id)] if thread_id else [],
```

Comment:
im deeply horrified we might summon satan at some point this is getting unholy

Reply:
whoopsie